packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.test.js

```diff
@@ -30,7 +30,7 @@ describe('<TrapFocus />', () => {
 
   it('should return focus to the children', () => {
     const { getByTestId } = render(
-      <TrapFocus open {...sharedProps}>
+      <TrapFocus {...defaultProps} open>
```

Comment:
For consistency with the other tests, cc @dmtrKovalenko.

Reply:
I think we are trying to get rid of `defaultProps` spreading in tests too?

Reply:
Yeah, I think that we are. The purpose here was:

- spread defaultProps first to have less priority
- use the same name as the other places, easier to refactor later on